src/components/SubscribeSection/index.tsx

```diff
@@ -17,10 +17,15 @@ const SubscribeSection: React.FC = () => (
       <div className={styles.subscribeContainer}>
         <SubscribeForm />
       </div>
+      <div className={styles.rssLink}>
+        <a href="/blog/rss.xml">
+          Keep updated on blog posts with our RSS Feed!
```

Comment:
Maybe just wrap the link around `our RSS Feed`? I feel like it makes it more clear what the link actually takes you to.

Reply:
That's actually how I had it just before pushing, then I expanded it on a whim because it seemed like the text was a bit useless. Changed because I was unsure either way so it's 1.5 to 0.5 in favor right now!